mesheryctl/internal/cli/root/config/config_test.go

```diff
@@ -265,6 +266,31 @@ func TestSetProvider(t *testing.T) {
 	}
 }
 
+func TestGetDisableOperator(t *testing.T) {
+	for _, test := range boolTests {
+		context := Context{"", "", "", nil, "", "", "", test}
+		got := context.GetDisableOperator()
+		want := test
+
+		if got != want {
+			t.Errorf("got %v want %v", got, want)
```

Comment:
it'd be good if we can change got/want to  received/expected

Reply:
@hexxdump The other tests follow the same got/want naming convention. It can be the same for consistency sake.